code/src/components/Submit.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+
+export const Submit = ({ thoughtsList, setThoughtsList }) => {
+  const [inputText, setInputText] = useState('');
+
+  // needs more work
+  const handleFormSubmit = (event) => {
+    event.preventDefault();
+    if (inputText.length < 5) {
+      console.log('too short')
+    } else { console.log('enough') }
+
+    fetch('https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts', {
+      method: 'POST',
+      body: JSON.stringify({ message: `${inputText}` }),
+      headers: { 'Content-Type': 'application/json' }
+    })
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error('Could not upload thought');
+        }
+        return response.json();
+      })
+      .then((newThought) => {
+        setThoughtsList([newThought, ...thoughtsList])
```

Comment:
Nice that you managed to get the spreading operator to work! I could not get that to work with ESLint